### PR TITLE
support cocoapods

### PR DESCRIPTION
**Pull request by dulingkang**

### CODE REVIEW

File: AnyCodable.podspec [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  s.description      = <<-DESC
+  Type-erased wrappers for Encodable, Decodable, and Codable values.
+  DESC
+
+  s.homepage         = 'https://github.com/Flight-School/AnyCodable'
+  # s.screenshots     = 'www.example.com/screenshots_1', 'www.example.com/screenshots_2'
+  s.license          = { :type => 'MIT', :file => 'LICENSE' }
+  s.author           = { 'mattt' => 'mattt' }
+  s.source           = { :git => 'https://github.com/dulingkang/AnyCodable.git', :tag => s.version.to_s }
```
Comment (marcosgriselli) [inline AnyCodable.podspec]: @dulingkang you're pointing to your forked repo rather than the original one.

**Comment by marcosgriselli** [comment]
Adding a CocoaPods example app seems too much fuzz. I'm not exactly sure what's the resolution of the CocoaPods issue when creating a pod with no example app (https://github.com/CocoaPods/pod-template/issues/204). Perhaps a Playground could be more useful if there's a need to show the library in action :)

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2018 dulingkang@163.com <dulingkang@163.com>
```
Comment (mattt) [inline LICENSE]: This LICENSE file appears to be a duplicate of the original LICENSE.md, which has the correct attribution. Please remove this file in your fork if you intend to continue using it.

**Comment by mattt** [comment]
Fixed by 24ffce0abd0858a7a62bea377bd93cc0289e855f

**Pull request closed without merging**